models/campaignModel.js

```diff
@@ -1,94 +1,97 @@
 const db = require('../database/dbConfig');
 
+
 const CampUpdate = require('./updateModel.js');
 const CampComments = require('./commentsModel.js');
 const CampLikes = require('./socialModel.js');
+const skills = require('../skilledImpactRequestsModel.js');
```

Comment:
nit: rename from `skills` to `skilledImpactRequests`/`SkilledImpactRequestModel`/etc.